.github/CODEOWNERS

```diff
@@ -15,4 +15,4 @@
 # This should make it easy to add new rules without breaking existing ones.
 
 # Global - These owners will be the default owners for everything in the repo.
-*           @ellajay @fleonard @habin-isa @jmuskett @josianed @mickaelzhang
+*           @ellajay @fleonard @habin-isa @josianed @mickaelzhang
```

Comment:
Wondering if maybe we could change this to the frontend group?

Reply:
the problem is that in the FE team there are a lot of BE and I want to avoid spamming them 😁